src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,36 @@
+package racing;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private Scanner scanner;
+
+    public InputView() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public int readCarCount(){
```

Comment:
```suggestion
    public int readCarCount() {
```

매의 눈 @_@

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.